javascript/javascript2/week2/hyf-bay/main.js

```diff
@@ -0,0 +1,114 @@
+console.log('Script loaded');
+
+const makerLists = (className, key) => {
+    const lists = document.createElement('li');
+    lists.classList.add(className);
+    lists.innerHTML = key;
+    return lists;
+}
+//Create some extra feature. Shopping cart.
+const items = [];
+const onClickBuy = (product) => {
+    const ulCart = document.querySelector('section.cart > ul');
+    const liName = document.createElement('li');
+    liName.innerHTML = product.name;
+    const liPrice = document.createElement('li');
+    liPrice.innerHTML = product.price;
+    ulCart.appendChild(liName);
+    ulCart.appendChild(liPrice);
+    items.push(product.price);
+
+    const totalPrice = items.reduce((a, c ) => (a += c));
+    const totalCartInfo = document.querySelector('div.total > p > span');
+    totalCartInfo.innerHTML = totalPrice;
+}
+
+const renderProducts = (products) => {
+    const ul = document.querySelector('section.products > ul');
+    ul.innerHTML = null;
+    products.forEach((object) => {
```

Comment:
I would suggest to pay more attention to how you name your variables...it might not be obvious to you at the moment but selecting good names for your variables, functions and classes helps you and your peer reviewers in understanding and checking your code.
```suggestion
    products.forEach((product) => {
```